Libraries/Components/StatusBar/NativeStatusBarManagerAndroid.js

```diff
@@ -12,13 +12,14 @@
 
 import type {TurboModule} from '../../TurboModule/RCTExport';
 import * as TurboModuleRegistry from '../../TurboModule/TurboModuleRegistry';
+import type {ProcessedColorValue} from '../../StyleSheet/StyleSheetTypes';
 
 export interface Spec extends TurboModule {
   +getConstants: () => {|
     +HEIGHT: number,
     +DEFAULT_BACKGROUND_COLOR: number,
   |};
-  +setColor: (color: number, animated: boolean) => void;
+  +setColor: (color: ProcessedColorValue, animated: boolean) => void;
```

Comment:
I think we were going to keep native modules as receiving `number` for now. Did this one get missed?

Reply:
Hmm, i guess so.  The @flow strict-local prevented it from checking outside of the file?  I removed this instance and flow checks still pass.

The RCTView.m definitely needs an if @available check.  Fixed.

The warnings in RCTUITextView.m don't seem to be related to the change -- I can't see how they could be: the `placeholderColor` @property did not changed nor did the `defaultPlaceholderColor()` function.